src/ray/raylet/pubsub/subscriber.cc

```diff
@@ -17,57 +17,36 @@
 namespace ray {
 
 void Subscriber::SubcribeObject(
-    const rpc::Address &owner_address, const ObjectID &object_id,
+    const rpc::Address &publisher_address, const ObjectID &object_id,
     SubscriptionCallback subscription_callback,
     SubscriptionFailureCallback subscription_failure_callback) {
   rpc::Address subscriber_address;
-  subscriber_address.set_raylet_id(self_node_id_.Binary());
-  subscriber_address.set_ip_address(self_node_address_);
-  subscriber_address.set_port(self_node_port_);
-  // Make a long polling connection if we never made the one with this owner for pubsub
-  // operations.
-  const auto owner_worker_id = WorkerID::FromBinary(owner_address.worker_id());
-  auto subscription_it = subscription_map_.find(owner_worker_id);
+  subscriber_address.set_raylet_id(subscriber_id_.Binary());
+  subscriber_address.set_ip_address(subscriber_address_);
+  subscriber_address.set_port(subscriber_port_);
```

Comment:
Nit: In interest of variables being defined in the smallest scope in which they are used, I'd usually vote for `subscriber_address` to be defined within the conditional, since we only need it when making a long-polling connection.